go.mod

```diff
@@ -2,6 +2,10 @@ module go.temporal.io/server
 
 go 1.18
 
+replace go.opentelemetry.io/contrib/instrumentation/github.com/gocql/gocql/otelgocql => github.com/temporalio/opentelemetry-go-contrib/instrumentation/github.com/gocql/gocql/otelgocql v0.32.1-0.20220715172623-ee0fd100b5cb
```

Comment:
If you decide to keep it, I would move them to the bottom of `go.mod`.

Reply:
> Why not just reference forks directly?

So that I can use the go.opentelemetry.io/x/y/z import path and if these changes make it upstream we can just remove the `replace` directive.

Reply:
This is actually a good point. We probably need to do it for all other custom forks.